### ISSUE TITLE
Nav Unification: Maintain menu item focus between Calypso & wp-admin

### ISSUE THREAD

**Issue by obenland** [Automattic/wp-calypso#51513]
Reported in p1617037801187100-slack-C01APA37DDW

### Steps to reproduce the behavior

1. Go to wordpress.com
2. Using the keyboard, navigate to "Feedback"
3. Once loaded, use the keyboard to navigate to "Comments"
4. Focus is reset and lost from the menu

#### What I expected to happen
That I could navigate to the Comments menu item from the Feedback menu item.

#### What actually happened
Focus got reset because it loaded a new page

#### Browser / OS version
Latest chrome

#### Level of impact
Affects all users who rely on keyboard navigation
